.changeset/pink-ladybugs-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': minor
```

Comment:
I considered this a breaking change because data in existing sqlite db files will be lost. Thinking on this more, I'm not sure if creating a sqlite db file was possible without these changes (see #6811). Maybe we can mark as `patch` in that case?

Reply:
I'd say patch in that case. Secondly I'm don't think we should give any guarantees for data persistence and migrations working when sqlite is used as database. It's primary there to make local development smoother but I'm happy to be wrong here :)

Reply:
I think we can make it patch too, in this case. But perhaps mention in the changeset that since the file names will change, users who use this feature will end up having their database start over empty.